thrift/src/main/thrift/atoms/media.thrift

```diff
@@ -0,0 +1,27 @@
+namespace * contentatom.media
+namespace java com.gu.contentatom.thrift.atom.media
+#@namespace scala com.gu.contentatom.thrift.atom.media
+
+typedef i64 Version
+
+enum AssetType {
+  YOUTUBE,
+  FACEBOOK,
+  PODCAST,
+  DAILYMOTION,
+  MAINSTREAM,
```

Comment:
I talked to @mchv about this earlier, and just wanted to add here that I agree about separating the type of the asset from the platform that is used to store it. I will make that change.